composer.json

```diff
@@ -22,15 +22,15 @@
         "laravel-zero/framework": "^10.0"
     },
     "require-dev": {
-        "vimeo/psalm": "dev-type-coverage as 5.20.0",
         "hyde/monorepo-dev-tools": "*",
         "hyde/realtime-compiler": "*",
         "hyde/testing": "*",
         "jetbrains/phpstorm-attributes": "^1.0",
         "laravel/tinker": "^2.7",
-        "pestphp/pest-plugin-type-coverage": "^2.8",
+        "pestphp/pest-plugin-type-coverage": "2.x-dev",
```

Comment:
We need https://github.com/pestphp/pest-plugin-type-coverage/commit/147f8b6d5dc180a756aa694d34f606050b823c71 which has not been released yet. Otherwise we get this:

```
  Too few arguments to function PHPStan\Analyser\NodeScopeResolver::__construct(), 26 passed in /home/runner/work/develop/develop/vendor/pestphp/pest-plugin-type-coverage/src/PHPStanAnalyser.php on line 80 and exactly 27 expected
```